source/process/feature-release.md

```diff
@@ -124,11 +124,11 @@ Day when Leads and PMs decide which major features are included in the release,
        - Post comments to the meta issue with approved fixes for the next RCs
        - Update download links and testing server links to the latest RCs
     - After build is cut, tweet announcement that RC1 is ready (see [example](https://community.mattermost.com/core/pl/tefx1ijyz7bs8mabuxmpq9f7pw))
-2. Logistics:
+    - Generate a list of contributors for Changelog
```

Comment:
By 'generate', is this using Hanzei's plugin?

Reply:
Yes, it's a slash command on a specific server.